articles/install-guide/vs-code.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Installing the Q# development environment for VS Code
+author: chgranad
+uid: microsoft.quantum.install.vs-code
+ms.author: christopher.granade@microsoft.com
+ms.date: 7/31/2018
+ms.topic: article
+---
+
+# Installing the Quantum Development Kit for Visual Studio Code #
```

Comment:
Instead of installing on Visual Studio Code. I would give instructions on how to do this from the command line. People get confused and think they need VS Code to work.

Then, at the end I would recommend installing VS Code and the Extension.